packages/primitives/src/ToggleGroup.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { toggleGroupAnatomy, ToggleGroup } from "@ark-ui/react";
+import { RecipeVariantProps, sva } from "@ndla/styled-system/css";
+import { JsxStyleProps } from "@ndla/styled-system/types";
+import { IconButton, IconButtonProps } from "./Button";
+import { createStyleContext } from "./createStyleContext";
+
+const toggleGroupRecipe = sva({
+  slots: toggleGroupAnatomy.keys(),
+  base: {
+    root: {
+      display: "flex",
+      flexDirection: "row",
+      padding: "3xsmall",
+      gap: "xxsmall",
+    },
+    item: {
+      "&[data-state='on']": {
+        backgroundColor: "surface.actionSubtle.active",
+        borderColor: "stroke.default",
+        border: "1px solid",
+      },
+    },
+  },
+  variants: {
+    variant: {
+      subtle: {
+        root: {
+          background: "surface.infoSubtle",
+          borderTopRadius: "xsmall",
+          borderColor: "stroke.subtle",
+          border: "1px solid",
+        },
```

Comment:
Dette virket veldig spesifikt for akkurat markdown-editoren. Tror ikke dette er anvendelig i ED. Kanskje det er greiere å bare style denne der den brukes.

Reply:
Det er designet fra Figma, så vi skal bare la den være tilnærmet plain?

Reply:
Den kan nok forbli fullstendig plain.